examples/go/example-datacenter.yaml

```diff
@@ -10,6 +10,31 @@ spec:
   cassandraImage: "gcr.io/cassandra-operator/cassandra:3.11.3"
   sidecarImage: "gcr.io/cassandra-operator/cassandra-sidecar:latest"
   imagePullPolicy: IfNotPresent
+  imagePullSecrets:
+    - name: regcred
+  backupSecretVolume:
+    secretName: backup-secret
+    #     type is a workaround for https://github.com/kubernetes/kubernetes/issues/68466
+    type: array
+    items:
+      - key: creds.json
+        path: creds.json
+  env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: "/etc/google/creds.json"
+  userSecretVolume:
+    secretName: dc1-user-secret
+  userConfigMapVolumeSource:
+    name: dc1-user-config
+    #     type is a workaround for https://github.com/kubernetes/kubernetes/issues/68466
+    type: array
+    items:
+      - key: cassandra_ssl
+        path: cassandra.yaml.d/003-ssl.yaml
+      - key: cqlshrc
+        path: cqlshrc
+      - key: install_cqlshrc
+        path: cassandra-env.sh.d/003-install-cqlshrc.sh
```

Comment:
This seems hacky... `cassandra-env.sh` is for setting up the JVM and its environment. Not other tools.

Reply:
sure, I'll think of a better usecase.

Reply:
removed, decided to use an environment variable and move it to the ~/.cassandra location with entry-point.